client.js

```diff
@@ -15,31 +15,35 @@ let client;
 
 async function lookup({ _, ...rest }) {
```

Comment:
Technically not on this context of this PR, but this function signature looks super strange to me. This looks like there is a single parameter that has a _ key that we're trying to filter out, using rest to gather all k-v pairs except for _. Is that correct? If you're not sure, we should probably open an issue to revisit.

Reply:
That is what my knowledge of javaScript gained during this project tells me too, but I think I just copied this from that original code that you created to insert/fetch without really thinking about it.
So bad of me :(
I'll open an issue about it.